src/graphs/graphs_traverse/Graph.java

```diff
@@ -0,0 +1,71 @@
+package graphs.graphs_traverse;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+public class Graph {
+    ArrayList<Node> nodes;
+    int[][] matrix;
+
+    Graph(int size){
+
+        nodes = new ArrayList<>();
+        matrix = new int[size][size];
+    }
+    public void addNode(Node node) {
+
+        nodes.add(node);
+    }
+    public void addEdge(int src, int dst) {
+
```

Comment:
same

Reply:
Resolved. Thank you for checking my code :)